changelog/8544.misc.md

```diff
@@ -0,0 +1 @@
+Catch and convert all `aio_pika` connection exceptions to subclass of `RasaException`.
```

Comment:
Only in event broker creation right?

Reply:
Yup, that's where I traced the error in the Sentry stacktrace. I'll add this note to the changelog.